### DISCUSSION TITLE
Nginx Proxy Manager with Fail2ban

### DISCUSSION THREAD

**Discussion by jubjubrsx** [tteck/Proxmox#3366] [Help Needed]
Has anyone been able to get fail2ban working with NPM with the tteck script? 

I've gone the docker route and it works with NPM 2.10.x but updated the docker to npm 2.11 and it doesnt work so not 100% sure if its the 2.11 version or what?

**Comment by Mellowlynx** [comment]
You should not expose the NPM UI to the internet, so that here is no need for f2b.
Use NPM's access list option to only allow your LAN to be able to visit the UI.

Updating a Docker container, destroys the old one, so you lose the changes you made in not mapped folders.
